SofaImGui/src/SofaImGui/AppIniFile.cpp

```diff
@@ -21,16 +21,24 @@
 ******************************************************************************/
 #include <SofaImGui/ImGuiGUIEngine.h>
 #include <sofa/helper/Utils.h>
+#include <sofa/gui/common/BaseGUI.h>
 #include "windows/Performances.h"
 #include "AppIniFile.h"
 
+
 using namespace sofa;
 
 namespace sofaimgui
 {
+    const std::string& getConfigurationFolderPath()
+    {
+        static const std::string configPath = sofa::gui::common::BaseGUI::getConfigDirectoryPath() + "/imgui";
```

Comment:
```suggestion
        static const std::string configPath = helper::system::FileSystem::append(sofa::gui::common::BaseGUI::getConfigDirectoryPath(), "imgui");
```
To ensure only one slash separator.